### PR TITLE
Setting url

### PR DESCRIPTION
**Pull request by EmmanuelEklipse**
Added ability to change backend url on the setting page https://ibb.co/N2WRnx1

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -68,7 +73,7 @@ android {
             // TODO: Add your own signing config for the release build.
             // Signing with the debug keys for now, so `flutter run --release` works.
             signingConfig signingConfigs.debug
-            signingConfig signingConfigs.release
+//            signingConfig signingConfigs.release
```
Comment (pshenmic) [inline android/app/build.gradle]: That should not be changed, probably accidentally pushed

File: lib/client.dart [hunk 2]
```diff
@@ -8,13 +8,12 @@ import 'dart:convert';
 import 'dart:io';
 
 class Client {
-  static final protocol = 'https';
-  static final domain = 'api.anypayx.com';
-  static final host = "$protocol://$domain";
+  static String protocol = 'https';
+  static String domain = 'anypay.pshenmic.dev';
```
Comment (pshenmic) [inline lib/client.dart]: Don't change default API

File: lib/client.dart [hunk 3]
```diff
@@ -8,13 +8,12 @@ import 'dart:convert';
...
+  static String host = "$protocol://$domain";
 
   static String humanize(str) {
-    if (str != null && str.length > 0)
-      return StringUtils.capitalize(str);
+    if (str != null && str.length > 0) return StringUtils.capitalize(str);
```
Comment (pshenmic) [inline lib/client.dart]: Revert all formatting

File: lib/generated_plugin_registrant.dart [hunk 4]
```diff
@@ -0,0 +1,27 @@
+//
+// Generated file. Do not edit.
```
Comment (pshenmic) [inline lib/generated_plugin_registrant.dart]: It says generated file, should it be pushed? What is it used for?

File: lib/package_info_helper.dart [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import 'package:package_info_plus/package_info_plus.dart';
```
Comment (pshenmic) [inline lib/package_info_helper.dart]: I think that file relates to the different PR, isn't it?

File: lib/routes/settings.dart [hunk 6]
```diff
@@ -79,86 +138,106 @@ class _SettingsPageState extends State<SettingsPage> {
...
+          onTap: () {
+            Navigator.pushNamed(context, '/settings/addresses').then((value) {
+              _successMessage = '';
+              _rebuild();
+            });
+          }),
     );
   }
 
+  // Widget _buildAboutUs(context) {
```
Comment (pshenmic) [inline lib/routes/settings.dart]: Remove out commented code

File: pubspec.yaml [hunk 7]
```diff
@@ -15,7 +15,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 # In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
 # Read more about iOS versioning at
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
-version: 2.3.1+18
+version: 2.3.1+19
```
Comment (pshenmic) [inline pubspec.yaml]: Don't change the version, it will be done before merging to staging branch

**Review by pshenmic — CHANGES_REQUESTED** [review]

File: pubspec.yaml (hunk 7 above)
Comment (EmmanuelEklipse) [reply pubspec.yaml]: This is 19 in the master branch, I think part of the issue we are having is that I pulled from master and you are asking me to push to develop and there are differences between develop and master

**Pull request closed without merging**